net.corluma.Corluma.yml

```
@@ -0,0 +1,25 @@
+app-id: net.corluma.Corluma
+runtime: org.kde.Platform
+runtime-version: '6.5'
+sdk: org.kde.Sdk
+command: corluma
+finish-args: 
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
```

```suggestion
  - --socket=x11
```

It doesn't actually work on Wayland.